encord/orm/project.py

```diff
@@ -135,8 +136,11 @@ def last_edited_at(self) -> datetime.datetime:
         return self["last_edited_at"]
 
     @property
-    def workflow_manager_uuid(self) -> UUID:
-        return self["workflow_manager_uuid"]
+    def workflow_manager_uuid(self) -> Optional[UUID]:
```

Comment:
This is kind of a breaking change, as the type checkers will get mad at it which could break customer builds.
So far the semantic was "you're just getting an exception on non-workflow project". Shall we keep it? 
Even though I really doubt that anyone is using this method, tbh.

Reply:
makes sense, returned this (with an extra docstring, and raising a "sematic" exception instead of just a `KeyError`)